miden-tx/src/tests/mod.rs

```diff
@@ -130,13 +122,7 @@ fn executed_transaction_account_delta() {
     let updated_map_value = [Felt::new(18), Felt::new(19), Felt::new(20), Felt::new(21)];
 
     // removed assets
-    let removed_asset_1 = Asset::Fungible(
-        FungibleAsset::new(
-            ACCOUNT_ID_FUNGIBLE_FAUCET_ON_CHAIN.try_into().expect("id is valid"),
-            FUNGIBLE_ASSET_AMOUNT / 2,
-        )
-        .expect("asset is valid"),
-    );
+    let removed_asset_1 = create_fungible_asset();
     let removed_asset_2 = Asset::Fungible(
         FungibleAsset::new(
             ACCOUNT_ID_FUNGIBLE_FAUCET_ON_CHAIN_2.try_into().expect("id is valid"),
```

Comment:
This could also probably be something like `FungibleAsset::mock(FUNGIBLE_ASSET_AMOUNT)`.